go.mod

```diff
@@ -11,12 +11,13 @@ require (
 	github.com/hashicorp/terraform-plugin-go v0.20.0
 	github.com/hashicorp/terraform-plugin-log v0.9.0
 	github.com/hashicorp/terraform-plugin-testing v1.6.0
-	github.com/stackitcloud/stackit-sdk-go/core v0.7.3
+	github.com/stackitcloud/stackit-sdk-go/core v0.7.4
 	github.com/stackitcloud/stackit-sdk-go/services/argus v0.8.0
 	github.com/stackitcloud/stackit-sdk-go/services/dns v0.7.0
 	github.com/stackitcloud/stackit-sdk-go/services/loadbalancer v0.8.0
 	github.com/stackitcloud/stackit-sdk-go/services/logme v0.8.0
 	github.com/stackitcloud/stackit-sdk-go/services/mariadb v0.8.0
+	github.com/stackitcloud/stackit-sdk-go/services/mongodbflex v0.8.0
```

Comment:
Why was there not a dependency on the `mongodbflex` module?

Reply:
There was on other line 69